index.html

```diff
@@ -5132,6 +5132,12 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MUST NOT be used to describe a verifiable credential with an <dfn>embedded proof</dfn>.
+  </p>
```

Comment:
```suggestion
  <p>
    If this media type is used to describe a verifiable credential any <dfn>embedded proof</dfn> MUST be ignored for the purposes of verification.
  </p>
```

Reply:
-1 to this. VCs and the three party model generally do not create a "control structure". Neither the issuer (nor this WG) get to tell a verifier what they can or cannot accept. I think media types are not the right way to accomplish the goals here.

If an API does not want to accept a particular field, it should say so / have a schema that rejects it, etc. Those are the right tools for that job.

Reply:
@dlongley are you ok with "should not" language for inclusion of an embedded proof?

Reply:
@mprorock,

>  are you ok with "should not" language for inclusion of an embedded proof?

No, I'm afraid not. I don't think it's the right use of media types. IMO, the language is unnecessarily restrictive, ineffectual, and likely to create more confusion and unnecessary complexity as a result.

However, I am not against more specific media types (other than that I think we should have a strong reason for every new media type we create). Nor am I against recommending that more specific media types be used if there's a desire to communicate that some data meets some subclass constraints. But that just seems to be general media type advice that shouldn't need to be said in a particular media type registration.

@msporny,

> IOW, the statement above says: "Use this 'unsecured' media type and embedded proofs together at your peril... DO use a more specific media type for expressing that the content is secured."

I think language like that is unnecessarily fearful. There's no reason to bring concepts like "use this as your own peril" into this. We don't need to create situations where people are afraid to consume certain data because of the media type that accompanies it. That further proliferates the wrong idea about media types, how to decide whether data is authentic (and what to do / not do about it), and how to work with data in the ecosystem generally, IMO.

In the three party model, there is no single party that tells other parties what to accept and then they just ... accept it. That's a core difference to some common two party models. There is no "Authorization Server" here. Instead, there is self-describing data that conforms to certain specs. Independent parties implement to those specs and when they see data that they understand -- they know how to consume it and what it means. If they don't understand it, they ignore it or reject it -- their choice. What they understand and are willing to accept is up to their business rules. They don't need to get together with other parties or have some special, elevated authorization party tell them what to do; that's just not how this technology works at all.

Reply:
@msporny This seems closer...
> > If `application/credential+ld+json` is specified, verifiers MAY ignore top-level embedded proofs. Implementers are strongly urged to use a more specific media type, such as `application/verifiable-credential+ld+json` if the credential is secured using an embedded proof.
> 

How about this variation:
```
If application/credential+ld+json is specified, verifiers MAY ignore top-level embedded proofs. Implementers SHOULD use a more specific media type, such as application/verifiable-credential+ld+json if the credential is secured using an embedded proof.
```

To your point re the registration side - I think you are correct there, in that I don't think the registration block is the right place for this.  I would like clear normative guidance about how to use the media types, and believe that the correct location for language to this effect does not need to be noted right inline below the table as it is done here, and probably could be located elsewhere in the spec.

Reply:
> If an API does not want to accept a particular field, it should say so / have a schema that rejects it, etc. Those are the right tools for that job.

@dlongley 

The v1 context says `proof` can be a member of the RDF type `VerifiableCredential`

- https://github.com/w3c/vc-data-model/blob/main/contexts/credentials/v1#L45

Perhaps the mistake we have made was to define `credential+ld+json` instead of `vc+ld+json`... since the concept of what a "credential" is seems to not have strong agreement, I thought it was obvious that a credential is not a verifiable credential and would not have an embedded or external proof, after reading the v1 spec... but it seems that there is no agreement on the interpretation of the v1 spec text:

<img width="800" alt="Screen Shot 2023-03-01 at 8 34 45 AM" src="https://user-images.githubusercontent.com/8295856/222171002-9bab1af6-4e40-4a6a-b513-6ba926d2ef96.png">

https://w3c.github.io/vc-data-model/#proofs-signatures

> What inspired the language in this PR? Was it from another media type registration? If so, which one?

@msporny confusion over what a "credential" is...  and wanting to define a media type so that the bytes for a representation of a "credential" can be secured and referred to consistently... As I noted above, perhaps there is no concept of `credential`, there is only `VerifiableCredential`... Certainly this is the case regarding the RDF interpretation of the core data model.

In either case, this dialog is emphasizing a critical flaw in the spec... it is clear the current language is being interpreted in drastically different ways by readers.

A few paths forward.

1. Remove `proof` from the core spec (rely on other specs, like data integrity to define it's presence in VCs, Capabilities, or other LD Objects)
2. Register `vc+ld+json` instead of `credential+ld+json`, since `proof` can be present in `vc+ld+json`
3. Remove the language from core spec about `credential` and just define `verifiable credential`

Reply:
> Perhaps the mistake we have made was to define credential+ld+json instead of vc+ld+json

Strong +1 to this. It's the point I was trying to convey in https://github.com/w3c/vc-data-model/issues/1044#issuecomment-1438778996

AFAIK there is not normative definition of what a `Credential` is. We only have a normative definition for a `VerifiableCredential`.

I am against 1, and in favor of doing 2 and 3.

Reply:
I'm not necessarily against what @andresuribe87 proposes (and the spirit of [his comment](https://github.com/w3c/vc-data-model/issues/1044#issuecomment-1438778996)) ... provided that we can make everything hang together.

Reply:
I should also note that @andresuribe87's [comment](https://github.com/w3c/vc-data-model/issues/1044#issuecomment-1438778996) I think somewhat captures what we tried do to simplify things in the 1.1 work.